kratos/geometries/geometry.h

```diff
@@ -540,7 +540,7 @@ class Geometry : public PointerVector<TPointType>
     @see WorkingSpaceDimension()
     @see LocalSpaceDimension()
     */
-    inline SizeType Dimension() const
+    virtual inline SizeType Dimension() const
```

Comment:
This is enough to block just in case

Reply:
Sure, as mentioned for the moment just for discussion. It doesnt compile like this.